### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by railton** [siriusdemon/Build-Your-Own-Face-Model#1]
Could you make the pre-trained model available?

Thanks for the tutorial

**Comment by siriusdemon** [comment]
Yes, I will upload a `facemobilenet` pre-trained model.

**Comment by jyuan1118** [comment]
> Could you make the pre-trained model available?
> 
> Thanks for the tutorial

I have uploaded two pretrained models.

face mobilenet   accuracy 96.8%
https://drive.google.com/drive/folders/1WbIUNF2EAOZNq9eWCTiD_5PtU2_hyh70?usp=sharing

Resnet accuracy: 96%
https://drive.google.com/drive/folders/1SyzQ9bPfUXu5EXUE7dlZmsD2pI0M9Os5?usp=sharing

**Comment by siriusdemon** [comment]
I have uploaded a face-mobile-net model in this repo

**Comment by Comedian1926** [comment]
Thank you for your work, can you tell me the hyperparameters you set, I am on the aligned data set (112, 96), after 20 epochs, the loss has not dropped around 15

**Comment by siriusdemon** [comment]
All in  [config.py](https://github.com/siriusdemon/Build-Your-Own-Face-Model/blob/master/recognition/config.py)

**Comment by Comedian1926** [comment]
> All in [config.py](https://github.com/siriusdemon/Build-Your-Own-Face-Model/blob/master/recognition/config.py)

thk
